src/main/resources/data/libcd/advancements/libcd/root.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
We'll want to add a condition mcmeta for this when we get to release time, so it doesn't get loaded in production environments.